tests/test_forum.py

```diff
@@ -262,30 +263,40 @@ async def test_edit_general_forum_topic(self, bot, forum_group_id):
         assert result is True, "Failed to edit general forum topic"
         # no way of checking the edited name, just the boolean result
 
-    async def test_close_and_reopen_general_forum_topic(self, bot, forum_group_id):
+    async def test_close_reopen_hide_unhide_general_forum_topic(self, bot, forum_group_id):
+        """Since reopening also unhides and hiding also closes, testing (un)hiding and
+        closing/reopening in different tests would mean that the tests have to be executed in
+        a specific order. For stability, we instead test all of them in one test."""
+
+        # We first ensure that the topic is open and visible
+        # Otherwise the tests below will fail
+        try:
+            await bot.reopen_general_forum_topic(chat_id=forum_group_id)
+        except BadRequest as exc:
+            if "Topic_not_modified" not in exc.message:
```

Comment:
maybe add a comment here saying that this is for when the topic is already opened